.github/DISCUSSION_TEMPLATE/Help.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Discussion
```

Comment:
What about naming it as _Help_? Since _Discussion_ it's an abstract word, _Discussion_ can be _Help_, _Idea_, _Feature Request_ etc.